Swagger does't work on localhost - Duplicate operationId: safes_delegates_delete

**Describe the bug**
When we start Transaction service on localhost using .env.local, Swagger doesn't works correctly

**To Reproduce**
Steps to reproduce the behavior:
1. Windows 10 WSL Ubuntu 20.04 / macOS 11
2. Clone reposotory
3. Create .env file from .env.local
4. Build docker images
5. Run docker compose
6. Open http://localhost:8000

And we got the follwing error in the console:
```
web_1        | 2021-09-20 18:12:35,309 [ERROR] [MainProcess] Internal Server Error: /
web_1        | Traceback (most recent call last):
web_1        |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
web_1        |     response = get_response(request)
web_1        |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 204, in _get_response
web_1        |     response = response.render()
web_1        |   File "/usr/local/lib/python3.9/site-packages/django/template/response.py", line 105, in render
web_1        |     self.content = self.rendered_content
web_1        |   File "/usr/local/lib/python3.9/site-packages/rest_framework/response.py", line 70, in rendered_content
web_1        |     ret = renderer.render(self.data, accepted_media_type, context)
web_1        |   File "/usr/local/lib/python3.9/site-packages/drf_yasg/renderers.py", line 35, in render
web_1        |     return codec.encode(data)
web_1        |   File "/usr/local/lib/python3.9/site-packages/drf_yasg/codecs.py", line 80, in encode
web_1        |     raise exc
web_1        | drf_yasg.errors.SwaggerValidationError: spec validation failed: {'ssv': 'Duplicate operationId: safes_delegates_delete'}
web_1        | 2021-09-20 18:12:36,785 MT::GET::::3007::500::/
```

```
Errors
Hide
 
Fetch errorInternal Server Error http://localhost:8000/?format=openapi
```
<img width="1047" alt="Screenshot 2021-09-20 at 22 26 09" src="https://user-images.githubusercontent.com/88528265/134062352-dcebcd72-7477-4d11-9ed4-2d88bef2ed19.png">


**Expected behavior**
Swagger should work on localhost


**Additional context**
Maybe something is wrong with our local configuration?

--- a/safe_transaction_service/history/views.py
+++ b/safe_transaction_service/history/views.py
@@ -470,7 +470,8 @@ class SafeDelegateListView(ListCreateAPIView):
         request.data['safe'] = address
         return super().post(request, address, **kwargs)
 
-    @swagger_auto_schema(responses={204: 'Deleted',
+    @swagger_auto_schema(operation_id='safes_delegates_delete_all',
+                         responses={204: 'Deleted',
                                     400: 'Malformed data',
                                     422: 'Invalid Ethereum address/Error processing data'})
     def delete(self, request, address, *args, **kwargs):

--- a/safe_transaction_service/notifications/tasks.py
+++ b/safe_transaction_service/notifications/tasks.py
@@ -127,7 +127,7 @@ def send_notification_task(address: Optional[str], payload: Dict[str, Any]) -> T
 
 
 @app.shared_task()
-def send_notification_owner_task(address: str, safe_tx_hash: str):
+def send_notification_owner_task(address: str, safe_tx_hash: str) -> Tuple[int, int]:
     """
     Send a confirmation request to an owner
 

